4/node.d.ts

```diff
@@ -1363,6 +1363,13 @@ declare module "dgram" {
 
     interface Socket extends events.EventEmitter {
         send(buf: Buffer, offset: number, length: number, port: number, address: string, callback?: (error: Error, bytes: number) => void): void;
+        send(buf: Buffer, offset: number, port: number, address: string, callback?: (error: Error, bytes: number) => void): void;
+        send(buf: Buffer, length: number, port: number, address: string, callback?: (error: Error, bytes: number) => void): void;
+        send(buf: Buffer, port: number, address: string, callback?: (error: Error, bytes: number) => void): void;
+        send(str: string, offset: number, length: number, port: number, address: string, callback?: (error: Error, bytes: number) => void): void;
+        send(str: string, offset: number, port: number, address: string, callback?: (error: Error, bytes: number) => void): void;
+        send(str: string, length: number, port: number, address: string, callback?: (error: Error, bytes: number) => void): void;
+        send(str: string, port: number, address: string, callback?: (error: Error, bytes: number) => void): void;
```

Comment:
LGTM. Would it be better to use a union? Reference: https://nodejs.org/api/dgram.html#dgram_socket_send_msg_offset_length_port_address_callback. E.g.

```ts
send(buf: string | Buffer | Array<any>, port: number, address: string, callback?: (error: Error, bytes: number) => void): void;
send(buf: string | Buffer, offset: number, length: number, port: number, address: string, callback?: (error: Error, bytes: number) => void): void;
```

(Not sure in the docs what array type it's mean to be.)